[MAP transformation] Update README.md (#9886)
openhab/openhab-addons: bundles/org.openhab.transform.map/README.md

# Map Transformation Service

> Transforms the input by mapping it to another string. It expects the mappings to be read from a file which is stored under the `transform` folder. The file name must end with the `.map` extension. 

Reviewer: ```suggestion
Transforms the input by mapping it to another string. It expects the mappings to be read from a file which is stored under the `transform` folder.
The file name must have the `.map` extension. 
```
